frontend/simple/views/ProposeMember.vue

```diff
@@ -1,165 +0,0 @@
-        this.candidateMembers.find(username => username === this.searchUser)
-      ) {
-        this.error = L('Invalid Username')
-        return
-      }
-      let member
-      try {
-        const contractId = await namespace.lookup(this.searchUser)
-        const state = await latestContractState(contractId)
-        member = { state, contractId }
-        this.searchUser = null
-        this.userErrorMsg = ''
-        this.proposed = false
-      } catch (err) {
-        console.log(err)
-        this.userErrorMsg = L('Invalid Username')
-      }
-      try {
-        // We need to have the latest mailbox attribute for the user
-        const mailbox = await backend.latestHash(member.state.attributes.mailbox)
-        const sentDate = new Date().toString()
-
-        // We need to post the invite to the users' mailbox contract
-        const invite = new Events.HashableMailboxPostMessage(
-          {
-            from: this.$store.getters.currentGroupState.groupName,
-            headers: [this.$store.state.currentGroupId],
-            messageType: Events.HashableMailboxPostMessage.TypeInvite,
-            sentDate
-          },
-          mailbox
-        )
-        // We need to determine if the invitation needs a vote
-        const groupLatest = await backend.latestHash(this.$store.state.currentGroupId)
-
-        // Create the Record for the group
-        const invited = new Events.HashableGroupRecordInvitation(
-          {
-            username: member.state.attributes.name,
-            inviteHash: '{lastActionHash}', // these are place holders to be added later
-            sentDate: '{actionDate}'
-          },
-          null
-        )
-        // Create the Proposal and add to it the record and invite actions
-        const proposal = new Events.HashableGroupProposal({
-          proposal: template(L('This is a Vote for {name} to become a member of {group}'),
-            {name: member.state.attributes.displayName || member.state.attributes.name, group: this.contract.groupName}
-          ),
-          // calculate the voting threshold from the group data
-          percentage: this.contract.memberApprovalPercentage * 0.01,
-          candidate: member.state.attributes.name,
-          actions: [
-           { contractId: member.state.attributes.mailbox, action: JSON.stringify(invite.toObject()) },
-           { contractId: this.$store.state.currentGroupId, action: JSON.stringify(invited.toObject()) }
-          ],
-          initiator: this.$store.state.loggedIn.name,
-          initiationDate: new Date().toString()
-        }, groupLatest)
-        await backend.publishLogEntry(this.$store.state.currentGroupId, proposal)
-        this.proposed = true
-      } catch (error) {
-        console.error(error)
-        // TODO: Create More descriptive errors
-        this.userErrorMsg = L('Failed to Propose Users')
-      }
-    },
-    remove (index) {
-      this.members.splice(index, 1)
-    }
-  }
-}
-</script>
```

Comment:
What happened to this file and why?

Reply:
"invitation and proposal views had ~10 lines of significant difference, so they moved to the same file now" => this is also handled by Invite.vue now